components/MessageModal/MessageModal.test.js

```diff
@@ -4,7 +4,6 @@ import userEvent from "@testing-library/user-event";
 
 test("displays MessageModal when value of 'isOpen' is true ", () => {
   render(<MessageModal isOpen={true}>test</MessageModal>);
-  //const button = screen.getByRole("button", { name: /Schließen/i });
   const button = screen.getByRole("button");
```

Comment:
I see that this works because there is only one button inside of the `MessageModal`. If there would be more, this seems to be a good case for using `getByTestId`.